docs/user-guide/services/index.md

```diff
@@ -424,6 +424,22 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
 an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is specified, but the
 cloud provider does not support the feature, the field will be ignored.
 
+#### Internal load balancer
+In a mixed environment sometimes it is necesarry to route traffic from services inside the same VPC (or equivivalent on non AWS platforms).
```

Comment:
Small grammar and spelling nits: "In a mixed environment`,` sometimes it is `necessary` to..."